client/src/components/EditAssociatedPositions.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react"
+import { Button } from "@blueprintjs/core"
+import { Modal, Row, Col, Grid } from "react-bootstrap"
+import PropTypes from "prop-types"
+import AssociatedPositions from "../pages/positions/AssociatedPositions"
+
+function EditAssociatedPositions({
```

Comment:
We (mostly) use the convention `const EditAssociatedPositions = ({ … }) => { … }`.